src/yb/server/default-path-handlers.cc

```diff
@@ -184,10 +184,20 @@ static void MemTrackersHandler(const Webserver::WebRequest& req, Webserver::WebR
   *output << "<table class='table table-striped'>\n";
   *output << "  <tr><th>Id</th><th>Current Consumption</th>"
       "<th>Peak consumption</th><th>Limit</th></tr>\n";
+  
+  int max_depth = INT_MAX;
+  string depth = FindWithDefault(req.parsed_args, "max_depth", "false");
+  if (depth != "false") {
```

Comment:
maybe make the default just empty string, so we don't hardcode a "false" string in the if check as well

Reply:
Sure